Sources/Task/ExistentialTask.swift

```diff
@@ -91,14 +91,16 @@ extension Task {
         self.init(future: base, progress: progress)
     }
 
+    private typealias _Self = Task<SuccessValue>
+
     /// Create a task whose `upon(_:_:)` method uses the result of `base`.
     ///
     /// If `base` is not a `Task`, `cancellation` will be called asynchronously,
```

Comment:
Given `base` has type `Task` in the parameter list, how would we end up with a `base` that isn't a `Task`?